ResearchKit/Common/ORKQuestionStepViewController.m

```diff
@@ -563,6 +563,7 @@ - (ORKSurveyAnswerCell *)answerCellForTableView:(UITableView *)tableView
     static dispatch_once_t onceToken;
     dispatch_once(&onceToken, ^{
         typeAndCellMapping = @{@(ORKQuestionTypeScale): [ORKSurveyAnswerCellForScale class],
+                               @(ORKQuestionTypeVerticalScale): [ORKSurveyAnswerCellForVerticalScale class],
```

Comment:
Instead of doing this, you can do a special case correction in the code below this to handle the vertical case.